### ISSUE TITLE
[1.7.0.

### ISSUE THREAD

**Issue by xabolcs** [playframework/play1#1399]
### Are you looking for help?

This is an issue tracker, used to manage and track the development of Play. It is not a support system and so it is not a place to ask questions or get help. If you're not sure if you have found a bug, or if you have a feature request, the best place to start is with either the [Discuss Play Forum](https://discuss.playframework.com) or [Stack Overflow](http://stackoverflow.com/questions/ask?tags=playframework).

### Operating System (Ubuntu 15.10 / MacOS 10.10 / Windows 10)

Use `uname -a` if on Linux.

### JDK (Oracle 1.8.0_72, OpenJDK 1.8.x, Azul Zing)

Paste the output from `java -version` at the command line.

### Library Dependencies

If this is an issue that involves integration with another system, include the exact version and OS of the other system, including any intermediate drivers or APIs i.e. if you connect to a PostgreSQL database, include both the version / OS of PostgreSQL and the JDBC driver version used to connect to the database.

### Expected Behavior

Please describe the expected behavior of the issue, starting from the first action.

1.
2.
3.

### Actual Behavior

Please provide a description of what actually happens, working from the same starting point.

Be descriptive: "it doesn't work" does not describe what the behavior actually is -- instead, say "the page renders a 500 error code with no body content."  Copy and paste logs, and include any URLs.  Turn on internal Play logging with `<logger name="play" value="TRACE"/>` if there is no log output.  

1.
2.
3.

### Reproducible Test Case

Please provide a PR with a failing test.  

If the issue is more complex or requires configuration, please provide a link to a project on Github that reproduces the issue.

**Comment by xabolcs** [comment]
**Cc** @jacol84

**Comment by jacol84** [comment]
@xabolcs  
thank you, for your vigilance 

I have sent another pull request with a correction this bug.

thank you,

**Comment by tomparle** [comment]
Nice catch+fix in such a short time !

**Comment by jacol84** [comment]
now it is working on WSL  [artefact ](https://github.com/playframework/play1/suites/5919617175/artifacts/202242348)

but when i run on win is not working

I have a problem with python and the win32pdh library
`
G:\tmp\playy>G:\tmp\playy\play-1.7.x-1.7.0-2-gd0c45c9d\play.bat run aaa
~        _            _
~  _ __ | | __ _ _  _| |
~ | '_ \| |/ _' | || |_|
~ |  __/|_|\____|\__ (_)
~ |_|            |__/
~
~ play! 1.7.x-1.7.0-2-gd0c45c9d, https://www.playframework.com
~
No module named 'win32pdh'
Traceback (most recent call last):
  File "G:\tmp\playy\play-1.7.x-1.7.0-2-gd0c45c9d\framework\pym\play\cmdloader.py", line 26, in load_core
    mod = load_python_module(name, self.path)
  File "G:\tmp\playy\play-1.7.x-1.7.0-2-gd0c45c9d\framework\pym\play\cmdloader.py", line 62, in load_python_module
    return imp.load_module(name, mod_desc[0], mod_desc[1], mod_desc[2])
  File "C:\Python36\lib\imp.py", line 235, in load_module
    return load_source(name, filename, file)
  File "C:\Python36\lib\imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 684, in _load
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "G:\tmp\playy\play-1.7.x-1.7.0-2-gd0c45c9d\framework\pym\play\commands\daemon.py", line 14, in <module>
    import win32pdh, win32pdhutil
ModuleNotFoundError: No module named 'win32pdh'
Traceback (most recent call last):
  File "G:\tmp\playy\play-1.7.x-1.7.0-2-gd0c45c9d\framework\pym\play\cmdloader.py", line 26, in load_core
    mod = load_python_module(name, self.path)
  File "G:\tmp\playy\play-1.7.x-1.7.0-2-gd0c45c9d\framework\pym\play\cmdloader.py", line 62, in load_python_module
    return imp.load_module(name, mod_desc[0], mod_desc[1], mod_desc[2])
  File "C:\Python36\lib\imp.py", line 235, in load_module
    return load_source(name, filename, file)
  File "C:\Python36\lib\imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 684, in _load
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "G:\tmp\playy\play-1.7.x-1.7.0-2-gd0c45c9d\framework\pym\play\commands\daemon.py", line 14, in <module>
    import win32pdh, win32pdhutil
`
on wsl

**Comment by tomparle** [comment]
Yes, try to add it on your system with 'pip install' to see if it does the trick

**Comment by jacol84** [comment]
thanks for the tips and tricks

`python -m pip install pywin32`

![image](https://user-images.githubusercontent.com/11667049/162250649-958daa3a-a507-468e-9d33-e7ae09a6ea33.png)

now it is working 
thank you @tomparle 


this is how it works, but rather this library should be included in the project

**Comment by xabolcs** [comment]
> ... but rather this library should be included in the project

Exactly!

The Python3 upgrade (#1394 and friends) seems to be incomplete! Lot of work was done but a few bits are missing.
